server/controllers/medical/patients/index.js

```diff
@@ -131,7 +145,15 @@ function generatePatientText(patient) {
 }
 
 /**
- * Returns details associated to a patient directly and indirectly
+ * @method detail
+ *
+ * @description
+ * Returns details associated to a patient directly and indirectly.
```

Comment:
I think it's supposed to be directly related things (like `dob`) and indirectly related things, like groups and such...

Note that we should update all these docs soon.  Once medical is feature complete, it will be a breeze.